.test-infra/jenkins/job_PreCommit_Java.groovy

```diff
@@ -30,6 +30,9 @@ PrecommitJobBuilder builder = new PrecommitJobBuilder(
       '^examples/java/.*$',
       '^examples/kotlin/.*$',
       '^release/.*$',
+    ],
+    excludePathPattners: [
```

Comment:
Do we need similar excludes for any of post-commits as well?

Reply:
I do not see any excludes on any of them. That may be valuable. At some point we are doing a lot of work that a dependency analysis should do ;_;